sdk/cognitiveservices/cognitiveservices-textanalytics/review/cognitiveservices-textanalytics.api.md

```diff
@@ -0,0 +1,346 @@
+    // (undocumented)
+    extractKeyPhrases(input: string[], language?: string, options?: ExtractKeyPhrasesOptions): Promise<ExtractKeyPhrasesResultCollection>;
+    // (undocumented)
+    extractKeyPhrases(input: MultiLanguageInput[], options?: ExtractKeyPhrasesOptions): Promise<ExtractKeyPhrasesResultCollection>;
+    // (undocumented)
+    recognizeEntities(input: string[], language?: string, options?: RecognizeEntitiesOptions): Promise<RecognizeEntitiesResultCollection>;
+    // (undocumented)
+    recognizeEntities(input: MultiLanguageInput[], options?: RecognizeEntitiesOptions): Promise<RecognizeEntitiesResultCollection>;
+    // (undocumented)
+    recognizePiiEntities(input: string[], language?: string, options?: RecognizePiiEntitiesOptions): Promise<RecognizeEntitiesResultCollection>;
+    // (undocumented)
+    recognizePiiEntities(input: MultiLanguageInput[], options?: RecognizePiiEntitiesOptions): Promise<RecognizeEntitiesResultCollection>;
+    // (undocumented)
+    singleAnalyzeSentiment(inputText: string, language?: string, options?: AnalyzeSentimentOptions): Promise<AnalyzeSentimentSuccessResult>;
+    // (undocumented)
+    singleDetectLanguage(input: string, countryHint?: string, options?: DetectLanguageOptions): Promise<DetectLanguageSuccessResult>;
+    // (undocumented)
+    singleExtractEntityLinking(inputText: string, language?: string, options?: ExtractEntityLinkingOptions): Promise<ExtractLinkedEntitiesSuccessResult>;
+    // (undocumented)
+    singleExtractKeyPhrases(inputText: string, language?: string, options?: ExtractKeyPhrasesOptions): Promise<ExtractKeyPhrasesSuccessResult>;
+    // (undocumented)
+    singleRecognizeEntities(inputText: string, language?: string, options?: RecognizeEntitiesOptions): Promise<RecognizeEntitiesSuccessResult>;
+    // (undocumented)
+    singleRecognizePiiEntities(inputText: string, language?: string, options?: RecognizePiiEntitiesOptions): Promise<RecognizeEntitiesSuccessResult>;
+}
+
+// @public
+export interface TextAnalyticsClientEntitiesLinkingOptionalParams extends coreHttp.RequestOptionsBase {
+    modelVersion?: string;
+    showStats?: boolean;
+}
+
+// @public
+export interface TextAnalyticsClientEntitiesRecognitionGeneralOptionalParams extends coreHttp.RequestOptionsBase {
+    modelVersion?: string;
+    showStats?: boolean;
+}
+
+// @public
+export interface TextAnalyticsClientEntitiesRecognitionPiiOptionalParams extends coreHttp.RequestOptionsBase {
+    modelVersion?: string;
+    showStats?: boolean;
+}
+
+// @public
+export interface TextAnalyticsClientKeyPhrasesOptionalParams extends coreHttp.RequestOptionsBase {
+    modelVersion?: string;
+    showStats?: boolean;
+}
+
+// @public
+export interface TextAnalyticsClientLanguagesOptionalParams extends coreHttp.RequestOptionsBase {
+    modelVersion?: string;
+    showStats?: boolean;
+}
+
+// @public (undocumented)
+export interface TextAnalyticsClientOptions {
```

Comment:
This should extend PipelineOptions from core-http.

Reply:
@bterlson ah, I see, we're extending instead of taking them as a parameter? I'm guessing we destructure the known props away then?

Reply:
Made this change for now.

Reply:
I don't follow your question - can you show me what you mean?

Reply:
this change: https://github.com/Azure/azure-sdk-for-js/pull/6433/commits/b7097b0e9e9417662abe36f3361a68b7c985e42e